src/polymer/js-utils.ts

```diff
@@ -46,27 +46,9 @@ export function toScannedPolymerProperty(
     }));
   }
 
-  let value;
-  if (babel.isClassMethod(node)) {
-    value = node;
-  } else {
-    value = node.value;
-  }
+  const value = babel.isObjectProperty(node) ? node.value : node;
 
-  let type;
-  if (value) {
-    type = closureType(value, sourceRange, document);
-  } else {
-    type = new Warning({
```

Comment:
Was this warning never created? Warnings are good things when we encounter invalid code, I'd like us to tend towards more of them rather than fewer!

Reply:
See note below.  This was a warning I added that turned out to not be a thing needed.